presto-orc/src/main/java/com/facebook/presto/orc/metadata/Stream.java

```diff
@@ -62,9 +62,9 @@ public StreamArea getStreamArea()
 
     private final Optional<Long> offset;
 
-    public Stream(int column, StreamKind streamKind, int length, boolean useVInts)
+    public Stream(int column, int sequence, StreamKind streamKind, int length, boolean useVInts)
```

Comment:
This is the place where the sequence id passed into the ColumnWriters.createColumnWriter should end up. It replaces the old constructor making it impossible to avoid passing the sequence id value.